devices/x15

```diff
@@ -17,7 +17,8 @@
     - setenv fdtfile am57xx-beagle-x15.dtb
     - setenv console ttyS2,115200n8
     - setenv mmcdev 1
-    - "setenv bootpart 1:9"
+    - part number mmc 1 userdata part_num
+    - "setenv bootpart 1:${part_num}"
```

Comment:
is the quotes needed ?

Reply:
I think it is. Otherwise yaml parsing fails. It was there before.

Reply:
right, forgot that.

LGTM!